lib/Notes/NotesAccordion/NotesAccordion.css

```diff
@@ -0,0 +1,3 @@
+a.new-button {
```

Comment:
is `a` needed?

Reply:
Unfortunately, it's needed to override specific selector